### ISSUE TITLE
Feature: Video Conversion

### ISSUE THREAD

**Issue by lafemmephile** [lafemmephile/core#10]
Purpose: Converting video files from one *video format* to another (e.g. `webm` to `mp4`).

Possible CLI design:
+ `video_path`: required file/directory path to videos selected for conversion (defaults to `CWD`)
   ```console
   femmephile convert <video_path>
   ```
+ `--format/-f`: video file format to convert target videos to (defaults to `mp4`)
  ```console
  femmephile convert -f webm <video_path>
  ```

**Comment by lafemmephile** [comment]
# References
+ https://zulko.github.io/moviepy/ref/VideoClip/VideoClip.html#moviepy.video.VideoClip.VideoClip.write_videofile

**Comment by lafemmephile** [comment]
Implemented with: 429c56d8147158554d8637be97c6a20b9e3725a6
